terminus/geometry/bounding_box.py

```diff
@@ -0,0 +1,38 @@
+from geometry.point import Point
+
+
+class BoundingBox(object):
+
+    def __init__(self, origin, corner):
+        self.origin = origin
+        self.corner = corner
+        self._normalize()
+
+    @classmethod
+    def from_boxes(cls, box_list):
+        try:
+            whole_merge = box_list[0]
+            for i in range(1, len(box_list)):
+                whole_merge = whole_merge.merge(box_list[i])
+            return whole_merge
+        except:
+            raise Exception('box_list should be a non empty list of bounding boxes')
```

Comment:
Although this is a very pythonic way of implementing it, there is a small issue: the `try/except` block will catch any exception, not just the case that the `box_list` is empty. Even though there are better ways to implement this (we can discuss them later) I would suggest doing something along the lines of:

```
if not box_list:
    raise ValueError("box_list can't be empty")
whole_merge = box_list[0]
for i in range(1, len(box_list)):
    whole_merge = whole_merge.merge(box_list[i])
￼return whole_merge
```